pkg/cmd/job/job.go

```diff
@@ -25,6 +25,7 @@ func NewCommand(ctx api.Context) *cobra.Command {
 	cmd.AddCommand(
 		newCmdJobAdd(ctx),
 		newCmdJobList(ctx),
+		newCmdJobKill(ctx),
 		newCmdJobUpdate(ctx),
```

Comment:
Please move it after `newCmdJobRun`.

Reply:
I have done that on the `job schedule` command which you will see on that PR for that.